src/conversions/hashbrown.rs

```diff
@@ -134,11 +134,13 @@ where
 mod tests {
     use super::*;
     use crate::types::IntoPyDict;
+    use std::hash::RandomState;
```

Comment:
Looks like it might be at a different path:

```suggestion
    use std::collections::hash_map::RandomState;
```